plugins/MauticFocusBundle/Model/FocusModel.php

```diff
@@ -363,12 +363,12 @@ protected function dispatchEvent($action, &$entity, $isNew = false, Event $event
     /**
      * @param bool $canViewOthers
      */
-    public function getStats(Focus $focus, $unit, \DateTime $dateFrom = null, \DateTime $dateTo = null, $dateFormat = null, $canViewOthers = true): array
+    public function getStats(Focus $focus, $unit, \DateTime $dateFrom, \DateTime $dateTo, $dateFormat = null, $canViewOthers = true): array
```

Comment:
This method signature change looks like BC to me. Is this required to fix this problem?

Reply:
the getStats function of the FocusModel class creates a ChartQuery object on line 369, whose parameters $dateFrom, $dateTo cannot have null values.

When you use GetStats with null $dateFrom and $dateTo variables, you get a typing error from the ChartQuery constructor.

That's why I changed it.

Reply:
Makes sense, thanks for explaining